### DISCUSSION TITLE
Using LoRAs

### DISCUSSION THREAD

**Discussion by gonzalu** [cubiq/ComfyUI_IPAdapter_plus#417] [Q&A]
Matteo thank you for IPAdapter and your fantastic tutorials.

I wanted to ask for your advice in using LoRAs in the workflow together with IPAdapter... where should it go in the pipeline, weights, prompting, etc. for best results. 

Thank you and Cheers!

**Comment by cubiq** [comment]
this is really too generic. there are hundreds of loras and the weight depends on the effect you are after. Generally speaking put the lora at the beginning just after the checkpoint loader

**Comment by gonzalu** [comment]
Thank you...
